main.go

```diff
@@ -47,6 +47,11 @@ func main() {
 			return &cmd.VersionCommand{
 				Ui:      ui,
 				Version: VersionString(),
+				BuildInfo: &cmd.BuildInfo{
```

Comment:
As mentioned we may not need to pass any extra arguments here, if we just source it all from the `runtime` package.